gulpfile.js

```diff
@@ -14,6 +14,57 @@ require('laravel-elixir-vue-2');
  */
 
 elixir(mix => {
-    mix.sass('app.scss')
-       .webpack('app.js');
+    mix
+        .styles(
+            [
+                'slick-carousel/slick/slick.css',
+                'slick-carousel/slick/slick-theme.css',
+                'eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css',
```

Comment:
That was my idea, obviously this can be tweaked for **your** needs. Really this is your domain.